File: test/fixtures/output/javascript/fetch/text-plain.js
const options = { method: 'POST', headers: { 'content-type': 'text/plain' }, body: 'Hello World' };

fetch('http://mockbin.com/har', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));

File: src/targets/ruby/faraday/fixtures/text-plain.rb
require 'faraday'

conn = Faraday.new(
  url: 'http://mockbin.com',
  headers: {'Content-Type' => 'text/plain'}
)

response = conn.post('/har') do |req|
  req.body = "Hello World"
end

puts response.status
puts response.body